index.js

```diff
@@ -33,6 +34,22 @@ module.exports = function (text, opts) {
 		}, padding);
 	}
 
+	if (typeof margin === 'number') {
+		margin = {
+			top: margin,
+			right: margin * 3,
+			bottom: margin,
+			left: margin * 3
+		};
+	} else {
+		margin = objectAssign({
+			top: 0,
+			right: 0,
+			bottom: 0,
+			left: 0
+		}, margin);
+	}
```

Comment:
Wonder if we could consolidate the `passing` & `margin` logic here somehow. Too much duplication.